build.gradle.kts

```diff
@@ -78,24 +83,50 @@ kotlin {
             }
         }
 
-        named("jvmMain") {
+        val jvmMain by getting {
+            dependsOn(commonMain)
```

Comment:
Can you double check this is necessary? Adding `dependsOn(commonMain)` should be automatic https://kotlinlang.org/docs/multiplatform-share-on-platforms.html#share-code-on-all-platforms

> You don't need to specify any `dependsOn` relations manually:
> 
> * For all platform-specific source sets that depend on the common source set, such as `jvmMain`, `macosX64Main`, and others

```suggestion
```